pom.xml

```diff
@@ -74,7 +74,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.1.1</cdap.version>
+    <cdap.version>6.2.0</cdap.version>
```

Comment:
shouldn't be changed. The plugin has 6.1.0 as a lower bound for the parent, which means the actual code dependency shouldn't be higher.